### PR TITLE
Link card to account

### PR DESCRIPTION
**Pull request by conorH22**
I need to create LinkCard method in UserService.  To be able to send a post request UserController.

### CODE REVIEW

**Comment by conorH22** [comment]
Need feedback on LinkCard method in UserService class and the LinkCard post request in UserController

File: src/main/java/euteam/cleanmachine/service/UserService.java [hunk 1]
```diff
@@ -72,6 +74,18 @@ public UserDto addUser(UserSignUpDto userSignUpDto) {
         return userDto;
     }
 
+    public UserDto LinkCard(NFCDto nfcDto) {
+        AuthItem  authlist;
+       // NFCCard card = new NFCCard();
+      //  card.setCardNumber(nfcDto.getCardNumber());
+        //UserDto userDto = new UserDto();
+      //  userDto.setAuthItemList();
```
Comment (Raul6469) [inline src/main/java/euteam/cleanmachine/service/UserService.java]: Looks great, but you need to get the user entity from the database with the username passed as a paramter, add the NFC Card to it's `AuthItem` list, and then save it again into the database with the `UserDao`.
```java
public UserDto LinkCard(String username, NFCDto nfcDto) {
  // ...
} 
```

**Comment by Raul6469** [comment]
@conorH22 I opened issue #39 with the rough idea on how to authenticate your requests in JavaScript if you want to check it out

**Comment by conorH22** [comment]
Great! I will have a look at [#39](https://github.com/eu-team/clean-machine/issues/39)

**Pull request merged**